test/e2e/chaos/basic.go

```diff
@@ -1266,6 +1268,156 @@ var _ = ginkgo.Describe("[Basic]", func() {
 		})
 	})
 
+	ginkgo.Context("[Sidecar Config]", func() {
+		var (
+			cmName      string
+			cmNamespace string
+		)
+
+		// delete the created config map in each test case
+		ginkgo.JustAfterEach(func() {
+			kubeCli.CoreV1().ConfigMaps(cmNamespace).Delete(cmName, &metav1.DeleteOptions{})
+		})
+
+		ginkgo.Context("[Template Config]", func() {
+
+			ginkgo.It("[InValid ConfigMap key]", func() {
+				ctx, cancel := context.WithCancel(context.Background())
+				cmName = "incorrect-key-name"
+				cmNamespace = chaosMeshNamespace
+				err := createTemplateConfig(ctx, cli, cmName,
+					map[string]string{
+						"chaos-pd.yaml": `name: chaosfs-pd
+selector:
+  labelSelectors:
+    "app.kubernetes.io/component": "pd"`})
+				framework.ExpectNoError(err, "failed to create template config")
+
+				err = wait.Poll(time.Second, 10*time.Second, func() (done bool, err error) {
+					dpi := kubeCli.AppsV1().Deployments(chaosMeshNamespace)
+					dep, err := dpi.Get(chaosControllerManager, metav1.GetOptions{})
```

Comment:
why do we need to check `controller-manager` here?

Reply:
In addition, how to check the key of the ConfigMap? I didn't find where to check it.

Reply:
These tests just want to make sure even the configuration is invalid, chaos mesh won't get crash.

What do you mean by `check the key of the ConfigMap`? This logic is in the chaos mesh controller.